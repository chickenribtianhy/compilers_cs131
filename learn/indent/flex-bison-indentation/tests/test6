stuff
  stuff
  stuff rest moar moar
