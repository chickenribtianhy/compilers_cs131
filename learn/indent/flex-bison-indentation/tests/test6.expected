stuff(1:1-5)
indent(2:3-3)
stuff(2:3-7)
stuff(3:3-7) with rest(3:8-22)
outdent(4:0-0)
